modules/sugar/src/test/ts/browser/CssTest.ts

```diff
@@ -39,8 +39,9 @@ UnitTest.test('CssTest', () => {
     Css.copy(c, c2);
     Css.copy(m, c2);
 
-    // NOTE: Safari, Firefox 71+ and Chrome 102+ seem to support styles for math ml tags, so the Css.copy(m, c2) clobbers the previous style
+    // NOTE: Safari, Firefox 71+ and Chromium 102+ seem to support styles for math ml tags, so the Css.copy(m, c2) clobbers the previous style
     const browser = PlatformDetection.detect().browser;
+    // Would cover other Chromium based browsers such as Chrome and Edge
```

Comment:
```suggestion
```
This is unnecessary and implied by `Chromium`. If you really want this it should be in the above comment, not here.